bootstrap/templates/addons/homepage/app/configmap.yaml.j2

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
Maybe use a configmapGenerator for this file instead?

Reply:
I think I previously used a configmapGenerator, but it didn't parse the vars, so I needed to drop all my secrets into cluster-secrets, which I didn't like (or put them plaintext in the annotations, which is not great).

We can go either way, but I found this one to be the most consistent.

Reply:
This is fine then